File: Livro-FundamentosDaProgramacaoDeComputadores/Capitulo3/Java/Exercicio6.java
/*6. Faça um programa que receba o salário base de um funcionário, calcule e mostre o salário a receber, sabendo-se que o funcionário tem gratificação de 5% sobre o salário base e paga imposto de 7% também sobre o salário base.*/

import java.util.Scanner;

public class Exercicio6 {
    public static void main(String[] args){

        Scanner scan = new Scanner(System.in);

        System.out.print("Digite o salário: ");
        double salarioBase = scan.nextDouble();

        double gratificacao = salarioBase * 5 / 100;
        double imposto = salarioBase * 7 / 100;
        double salario = salarioBase + gratificacao - imposto;

        System.out.printf("\nO salário a receber é de %.1f.%n", salario);

        scan.close();
    }
}

File: Livro-FundamentosDaProgramacaoDeComputadores/Capitulo3/Python/Exercicio6.py
#6. Faça um programa que receba o salário base de um funcionário, calcule e mostre o salário a receber, sabendo-se que o funcionário tem gratificação de 5% sobre o salário base e paga imposto de 7% também sobre o salário base.

salarioBase = float(input("Digite o salario: "))

gratificacao = salarioBase * 5 / 100
imposto = salarioBase * 7 / 100
salario = salarioBase + gratificacao - imposto

print("\nO salario é ", salario)